bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableClient.scala

```diff
@@ -148,11 +150,14 @@ case object RemoteBackend extends CachedBackend {
 
   private def format(a: Angle)= f"${a.toDegrees}%4.03f"
 
-  def queryParams(qs: CatalogQuery): Array[NameValuePair] = Array(
-    new NameValuePair("CATALOG", qs.catalog.id),
-    new NameValuePair("RA", format(qs.base.ra.toAngle)),
-    new NameValuePair("DEC", f"${qs.base.dec.toDegrees}%4.03f"),
-    new NameValuePair("SR", format(qs.radiusConstraint.maxLimit)))
+  protected [votable] def queryParams(q: CatalogQuery): Array[NameValuePair] = q match {
+    case qs: ConeSearchCatalogQuery => Array(
+      new NameValuePair("CATALOG", qs.catalog.id),
+      new NameValuePair("RA", format(qs.base.ra.toAngle)),
+      new NameValuePair("DEC", f"${qs.base.dec.toDegrees}%4.03f"),
+      new NameValuePair("SR", format(qs.radiusConstraint.maxLimit)))
+    case _                          => Array.empty
+  }
```

Comment:
Yes, we have a very old version of `http-client` from apache commands that uses java Arrays